server/routes/koala.router.js

```diff
@@ -23,6 +23,20 @@ pool.on('error', (err, client) => {
 });
 
 // GET
+koalaRouter.get('/', (req, res) => {
```

Comment:
I think the route should match the url in ajax -  '/koalas'

Reply:
the router uses /koalas to send the get request to the koala.routers.js